src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -79,4 +84,39 @@ public CarinaDescriptionPageBase login() {
         return clickLoginBtn();
     }
 
+    @Override
+    public boolean isPageOpened() {
+        return nameInputField.isElementPresent() &&
+               passwordInputField.isElementPresent() &&
+               maleRadioBtn.isElementPresent() &&
+               femaleRadioBtn.isElementPresent() &&
+               privacyPolicyCheckbox.isElementPresent() &&
+               loginBtn.isElementPresent();
+    }
+
+    @Override
+    public boolean isMaleSexChecked() {
+        return maleRadioBtn.isChecked();
+    }
+
+    @Override
+    public boolean isFemaleSexChecked() {
+        return femaleRadioBtn.isChecked();
+    }
+
+    @Override
+    public boolean isPrivacyPolicyChecked() {
+        return privacyPolicyCheckbox.isChecked();
+    }
+
+    @Override
+    public boolean isNamePrinted(){
+        return nameInputField.getSize()!=null;
```

Comment:
verify text which you type before, its will be better
Example: 
public boolean isTypedNamePresent(String name) {
return name.equals(nameInputField.getText);

Reply:
done